server/src/test/java/org/elasticsearch/index/engine/LuceneChangesSnapshotTests.java

```diff
@@ -202,7 +200,7 @@ public void testUpdateAndReadChangesConcurrently() throws Exception {
         CountDownLatch readyLatch = new CountDownLatch(followers.length + 1);
```

Comment:
NIT: I guess we could just set this to count `1` now and only keep the count down in the main thread? (the countdowns in the `run` method seem redundant now that it's not doing anything before invoking `countDown` right?)

Reply:
We use "readLatch" to ensure that all threads are ready (i.e. scheduled) before testing.

Reply:
Ah right, that makes sense -> sorry for the noise :)

Reply:
no worries @original-brownbear :)